CHANGELOG-0.4.md

```diff
@@ -1,10 +1,14 @@
 # Changelog 0.4
 
-## [0.4.2] 2019-11-TOCHANGE
+## [0.4.2] 2019-11-15
```

Comment:
Create draft release since this date will be different.

Reply:
Fixed